static/js/containers/VideoDetailPage.js

```diff
@@ -95,7 +96,13 @@ class VideoDetailPage extends React.Component {
     this.uploadVideoSubtitle();
   };
 
+  updateCorner = async (corner: string) => {
```

Comment:
I don't think this needs to be an async function since `dispatch(updateVideoJsSync(corner))` isn't returning a `Promise`. Can you remove the async here and change the type signature to return `void`?

Reply:
OK